app/config/parameters.yml.dist

```diff
@@ -38,3 +38,8 @@ parameters:
     # Array of authorized superdesk servers
     superdesk_servers:
         - superdesk.dev
+
+    # Array of trusted proxies
+    trusted_proxies:
+        - 192.0.0.1
+        - 10.0.0.0/8
```

Comment:
maybe let's make it `~` by default? Because it will not work if the IP of the reverse proxy address is changing all the time